frontends/p4/fromv1.0/v1model.h

```diff
@@ -20,7 +20,14 @@ limitations under the License.
 #include "lib/cstring.h"
 #include "frontends/common/model.h"
 #include "frontends/p4/coreLibrary.h"
+#include "frontends/p4/methodInstance.h"
 #include "ir/ir.h"
+#include "lib/json.h"
+#include "backends/bmv2/sharedActionSelectorCheck.h"
+
+namespace BMV2 {
+class Backend;
```

Comment:
Please, do not mix the P4-14 model with the back-end. They are completely separate.